service/controller/v18/resource/endpoint/spec.go

```diff
@@ -1,6 +1,14 @@
 package endpoint
 
+import (
+	corev1 "k8s.io/api/core/v1"
+)
+
+// TODO Addresses and IPs are basically the same and we only need Addresses in
+// fact. The whole thing could be refactored and simplified a little bit later.
```

Comment:
How about TODO issue? And why not just using one now?

Reply:
I will create an issue.

Reply:
See https://github.com/giantswarm/giantswarm/issues/5154.